service_mesh/v2x/ossm-reference-smcp.adoc

```diff
@@ -3,9 +3,29 @@
 include::modules/ossm-document-attributes.adoc[]
 :context: ossm-reference
 
+You can customize your {ProductName} by modifying the default `ServiceMeshControlPlane` (SMCP) resource or by creating a completely custom SMCP resource. This reference section documents the configuration options available for the SMCP resource.
+
 include::modules/ossm-cr-example.adoc[leveloffset=+1]
 
-include::modules/ossm-cr-threescale.adoc[leveloffset=+1]
+== spec parameters
+
+include::modules/ossm-cr-general.adoc[leveloffset=+2]
+
+include::modules/ossm-cr-profiles.adoc[leveloffset=+2]
+
+For information about creating profiles, see the xref:../../service_mesh/v2x/ossm-profiles-users.adoc#ossm-control-plane-profiles_ossm-profiles-users[Creating control plane profiles].
+
+For more detailed examples of security configuration, see the xref:../../service_mesh/v2x/ossm-security.adoc#ossm-security-mtls_ossm-security[Security section].
+
+include::modules/ossm-cr-techPreview.adoc[leveloffset=+2]
+
+include::modules/ossm-cr-tracing.adoc[leveloffset=+2]
+
+include::modules/ossm-cr-version.adoc[leveloffset=+2]
+
+include::modules/ossm-cr-threescale.adoc[leveloffset=+2]
+
+include::modules/ossm-cr-status.adoc[leveloffset=+1]
 
 [id="additional-resources_ossm-reference"]
 == Additional resources
```

Comment:
For future, consider updating links so that the names match the titles of the section
e.g.
** xref:../../service_mesh/v2x/ossm-security.adoc#ossm-security-mtls_ossm-security[Mutual Transport Layer Security (mTLS)]

also, one more instance of `ServiceMeshControlPlane` that needs "object" or something after it below 😉 but like I said, can be done another time, I know it's out of scope of this PR for now